src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -1073,6 +1073,10 @@ def _configure_security_policy_storage_params(arg_ctx):
                    options_list=['--capacity', '-c'],
                    help='The capacity of the managed instance in vcores.')
 
+        c.argument('collation',
+                   options_list=['--collation'],
```

Comment:
Using `options_list` here is unnecessary. The cli option / parameter will be automatically built from the function dest parameter 'collation'. Use `options_list` when you want the cli parameter to have a different name or when you want to add a short parameter as in line 1074 above.